scripts/Dockerfile

```diff
@@ -5,38 +5,55 @@
 
 # $ docker run --init --rm --name bytebase --publish 8080:8080 --volume ~/.bytebase/data:/var/opt/bytebase bytebase/bytebase
 
-FROM node:18 as frontend
+FROM --platform=${BUILDPLATFORM} node:18 as frontend
 
+ARG NVM_MIRROR=https://mirrors.ustc.edu.cn/node/
+ARG NODE_MIRROR=https://mirrors.ustc.edu.cn/node/
+ARG NPM_REGISTRY=https://npmreg.proxy.ustclug.org/
```

Comment:
ought to change build script to accept mirrors settings, and shouldn't have default value to CN country settings